packages/create-astro/src/index.ts

```diff
@@ -83,19 +77,22 @@ function isValidProjectDirectory(dirPath: string) {
 	return conflicts.length === 0;
 }
 
-const { version } = JSON.parse(
-	fs.readFileSync(new URL('../package.json', import.meta.url), 'utf-8')
-);
-
 const FILES_TO_REMOVE = ['.stackblitzrc', 'sandbox.config.json', 'CHANGELOG.md']; // some files are only needed for online editors when using astro.new. Remove for create-astro installs.
 
 // Please also update the installation instructions in the docs at https://github.com/withastro/docs/blob/main/src/pages/en/install/auto.md if you make any changes to the flow or wording here.
 export async function main() {
 	const pkgManager = detectPackageManager()?.name || 'npm';
+	const [username, version] = await Promise.all([getName(), getVersion()]);
+	const skipHouston = Object.entries(args).filter(([k]) => k !== '_').length > 0;
```

Comment:
is there a reason for this kind of hidden `_` arg? If so, a comment would be good for future reference. If not, what about something more clear like `--simple-ui` or `--disable-houston`?

Reply:
Agreed! Made this explicit with a `--skip-houston` flag.